cmd/admission/main.go

```diff
@@ -18,7 +18,9 @@ func main() {
 
 	ah := &admission.AdmissionHook{}
 
-	flagset.StringVarP(&ah.ProviderURL, "provider-url", "", "", "URL of OVN manager (e.g. Neutron) API server")
+	flagset.StringVarP(&ah.ProviderAuthURL, "provider-auth-url", "", "", "Full Auth URL for OVN Manager")
```

Comment:
Is there a reason to not to call it provider-keystone-url? Just to make it clear. Also mention it the rest of introduced parameters.